### PR TITLE
Switch to ember-common-tags

### LINKED ISSUE #21: switch to ember-common-tags
I released an [ember-common-tags](https://github.com/bgentry/ember-common-tags) addon. This should remove the need for browserify in this addon's dev dependencies, and will clean up imports (i.e. `import { stripIndent } from 'common-tags'` instead of `npm:common-tags` w/ manual destructuring).

It should be super easy to switch over this addon and remove all traces of the old way. Anybody from the community want to take a crack at it to get on the contributor list? :pray:

### PR DESCRIPTION
**Pull request by viniciussbs**
Closes #21

### CODE REVIEW

**Comment by viniciussbs** [comment]
The tests are passing locally. Any thoughts on that?

File: tests/dummy/app/app.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
 // ember-browserify requires every package to be imported somewhere in the `app` tree
-import 'npm:common-tags';
+import 'common-tags';
```
Comment (dfreeman) [inline tests/dummy/app/app.js]: You should be able to get rid of this line (and the `ember-browserify` dependency in `package.json`) with this change :)

Comment (viniciussbs) [inline tests/dummy/app/app.js]: I've seen that we are using `yarn` here. I've updated the PR with a new `yarn.lock`. Let's see if it builds.

Comment (viniciussbs) [inline tests/dummy/app/app.js]: I've removed the line and the `ember-browserify` dependency and the tests are passing here. I'll wait the current build to finish and then I'll rebase.

**Comment by bgentry** [comment]
Thanks @viniciussbs!

**Pull request merged**